src/transformers/models/rag/modeling_rag.py

```diff
@@ -1089,9 +1089,6 @@ def __init__(
     def set_retriever(self, retriever: RagRetriever):
         self.rag.retriever = retriever
 
-    def adjust_logits_during_generation(self, logits, cur_len, max_length):
```

Comment:
<3

Reply:
We might have to update RAG's config to be sure that this works correctly - did you run all the RAG slow tests as well?